### DISCUSSION TITLE
IEMOCAP wav2vec2 model not training

### DISCUSSION THREAD

**Discussion by Janardhan2131** [speechbrain/speechbrain#1685] [Q&A] [answered]
I am trying to train the model IEMOCAP wav2vec2 model using the  following command 

%cd /content/speechbrain/recipes/IEMOCAP/emotion_recognition
!python train_with_wav2vec2.py hparams/train.yaml --data_folder = "/content/drive/MyDrive/IEMOCAP_full_release"

But It just running for 1 sec and not training at all and not running any epochs
![image](https://user-images.githubusercontent.com/84712492/200156522-58bdce6d-40b6-4b18-b62c-5d8e8fe21ff4.png)

**Comment by Janardhan2131** [comment, marked as answer]
Issue is solved.

**Reply by Adel-Moumen** [reply]
For the record, the issue was due to python 3.7, instead, the user needs to use python 3.8.
